aquality-tracking-cucumber5-jvm/src/main/java/aquality/tracking/integrations/cucumber5jvm/TestCaseResultParser.java

```diff
@@ -0,0 +1,50 @@
+package aquality.tracking.integrations.cucumber5jvm;
+
+import aquality.tracking.integrations.core.FinalResultId;
+import io.cucumber.plugin.event.Result;
+import lombok.Data;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+import static java.lang.String.format;
+
+class TestCaseResultParser {
+
+    private final Result result;
+
+    TestCaseResultParser(Result result) {
+        this.result = result;
+    }
+
+    TestCaseResult parse() {
+        TestCaseResult testCaseResult = new TestCaseResult();
+        switch (result.getStatus()) {
+            case FAILED:
+                testCaseResult.setFinalResultId(FinalResultId.FAILED);
+                testCaseResult.setFailReason(formatErrorMessage(result.getError()));
+                break;
+            case PASSED:
+                testCaseResult.setFinalResultId(FinalResultId.PASSED);
+                break;
+            case PENDING:
+            case SKIPPED:
+                testCaseResult.setFinalResultId(FinalResultId.PENDING);
+                testCaseResult.setFailReason("Test skipped");
```

Comment:
I am not sure, a point just to discuss, I think we could provide some reason why a test is skipped and it will not equal "Test skipped"

Reply:
When import skips the test?

Reply:
yes, 
I just think, could we have another FailReason In some cases or not?

Reply:
As far as I remember, Aquality Tracking sets PASSED, FAILED, PENDING, NOT_EXECUTED fail resolutions.

Reply:
When we want to skip test programmatically, we throw PendingException which means PENDING faildReasonId

Reply:
yes, that what I meant, and in this case, fail reason will be "Test skipped" in aquality tracking.
But I think it would be better to see a message from  PendingException , but I am not sure it is possible

Reply:
It is possible. I fixed it.